networkx/algorithms/approximation/tests/test_coalition_formation.py

```diff
@@ -0,0 +1,99 @@
+import math
```

Comment:
Add a comment at the beginning of the file, citing the paper, the authors and the programmer.

Reply:
Its already there, or you mean that I need to move it to the first lines ?